pyproject.toml

```diff
@@ -105,7 +105,7 @@ lint.select = [
   "ASYNC", # flake8-async
 ]
 # Ignore rules which conflict with ruff formatter.
-lint.ignore = ["COM812", "ISC001",]
+lint.ignore = ["COM812", "ISC001", "RUF100"]
```

Comment:
RUF100 shouldn't be a problem if ruff version in pre-commit, github workflow and requirements dev match. Usually pre-commit automatically removes unused noqa.